templates/publisher/account-snaps.html

```diff
@@ -0,0 +1,166 @@
+{% extends "_layout.html" %}
+
+{% block title %}
+My snaps — Linux software in the Snap Store
+{% endblock %}
+
+{% block content %}
+<div class="p-strip is-shallow u-no-padding--bottom">
+  <div class="row">
+    <div class="col-12">
+      <h1 class="p-heading--three u-float--left">My snaps</h1>
+      <a href="https://docs.snapcraft.io/build-snaps/" class="p-link--external u-float--right">How to create a snap</a>
+    </div>
+  </div>
+  <div class="row u-no-margin--top">
+    <div class="col-12">
+      <hr {% if not snaps %}class="u-no-margin--bottom" {% endif %}/>
+    </div>
+  </div>
+</div>
+{% if not snaps %}
+  <div class="p-strip--light is-shallow">
+    <div class="row u-no-margin--top">
+      <div class="col-12">
+        <h2 class="p-heading--four">Get started&hellip;</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-6">
+        <h3 class="p-heading--five">'Hello world' with snapcraft</h3>
+        <p>Try this 15 minute tutorial to get to know the basics of the snap format and snapcraft tool.</p>
+        <p><a href="https://tutorials.ubuntu.com/tutorial/create-your-first-snap#0" class="p-button--neutral p-link--external">Try the tutorial</a></p>
+      </div>
+      <div class="col-6">
+        <h3 class="p-heading--five">Publish a snap with the tools you know</h3>
+        <p>Electron, Python, Go and many more examples of how to 'snap' an application using familiar languages.</p>
+        <p><a href="https://docs.snapcraft.io/build-snaps/languages" class="p-button--neutral">Choose a language guide</a></p>
+      </div>
+    </div>
+  </div>
+  <div class="p-strip is-shallow">
+    <div class="row p-divide">
+      <div class="col-6 p-divider__block">
+        <h3 class="p-heading--five">Snapcraft documentation</h3>
+        <p>All you need to know about snaps, snapcraft and everything in between.</p>
+        <p><a href="https://docs.snapcraft.io">Read the docs &rsaquo;</a></p>
+      </div>
+      <div class="col-6 p-divider__block">
+        <h3 class="p-heading--five">Join the community</h3>
+        <p>Request features, troubleshoot and generally find all the latest talk about snaps, snapd and snapcraft.</p>
+        <p><a href="https://forum.snapcraft.io">Go to the focum &rsaquo;</a></p>
```

Comment:
Typo: `focum`

Reply:
Is this going to have just one entry? If so, This layout is not appropriate. instead, Things to do next should be rephrased to: 
Next, would you like to improve listings in sotre? [   Go to Market   ]